testing/geoclue/APKBUILD

```diff
@@ -1,34 +1,42 @@
+# Contributor: Leo <thinkabit.ukim@gmail.com>
 # Maintainer: 
 pkgname=geoclue
-pkgver=2.4.7
-pkgrel=2
+pkgver=2.5.2
+pkgrel=0
 pkgdesc="dbus geolocation service"
 url="https://www.freedesktop.org/wiki/Software/GeoClue/"
 arch="all"
-license="LGPL"
-depends=""
-depends_dev=""
-makedepends="libsoup-dev json-glib-dev modemmanager-dev avahi-dev intltool gobject-introspection-dev"
-install=""
+license="LGPL-2.1-or-later"
+makedepends="
+	meson
+	libsoup-dev
+	json-glib-dev
+	modemmanager-dev
+	avahi-dev
+	intltool
+	gobject-introspection-dev
+	libnotify-dev"
```

Comment:
```suggestion
	libnotify-dev
	"
```

Reply:
i prefer keeping the ending quote where it ends.

Reply:
It makes for cleaner diffs when it has a line to itself.